### PR TITLE
whitepaper.md [FULL TAGALOG TRANSLATION][FOR REVIEW]

### PR DESCRIPTION
**Pull request by cordeta**
Part 1 : From line 1 to 62 
A total of 1112 translated words from ENG to TL

Part 2 : From line 63 to 113 
A total of 454 translated words from ENG to TL

### CODE REVIEW

**Comment by agentzero1** [comment]
Why there's no english File?

File: tl-tl/node/whitepaper.md [hunk 1]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
-Para malabanan ang mga potensyal na kamalian, ang sistemang blockchain ay nangangailangan ng mabisang mekanismong pangkonsensus upang matiyak na ang bawat noda ay may kopya ng mga kilalang bersyon ng kabuuang pagtatala. Mekanismong tadisyunal na toleransiya ng kamalian na hinggil sa ilang problema ay hindi maaaring may ganap na kakayahang harapin ang isyu na ang sistemang blockchain at pamamahagi na hinaharap nito. isang unibersal na toleransiya ng kamalian na solusyong panglunas ang kinakailangan.
+Para malabanan ang mga potensyal na kamalian, ang sistemang blockchain ay nangangailangan ng mabisang mekanismong pangkonsensus upang matiyak na ang bawat node ay may kopya ng mga kilalang bersyon ng kabuuang pagtatala. Mekanismong tadisyunal na toleransiya ng kamalian na hinggil sa ilang problema ay hindi maaaring may ganap na kakayahang harapin ang isyu na ang sistemang blockchain at pamamahagi na hinaharap nito. isang unibersal na toleransiya ng kamalian na solusyong panglunas ang kinakailangan.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: Ang mekanismong tradisyunal?

File: tl-tl/node/whitepaper.md [hunk 2]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
 mekanismong pruweba-ng-trabaho[1], inempleyo ng Bitcoin, bagkus ay tinatalakay nito ang isyu ng may katalinuhan. Ngunit ito ay sadyang may kamahalan talaga, halimbawa, mahal na presyo ng kuryenteng bayarin at mataas na konsampsyon ng enerhiya. Dagdag pa rito, dahil sa presensya ni Bitcoin, ang mga bagong blockchain ay kailangang maghanap ng ibang hashing na mga algoritmo, ng sa ganun ay maiwasan ang komputisyunal sa pag-atake sa kanila. halimbawa, inadopt ni Litcoin ang SCRYPT, kesa sa SHA256 ni Bitcoin.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: konsampsyon to paggamit

Comment (severinolorillajr) [reply tl-tl/node/whitepaper.md]: komputisyonal?

File: tl-tl/node/whitepaper.md [hunk 3]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
 ## Modelo ng Sistema 
 
-Ang blockchain ay isang sistema ng ipinamahaging pagtatala na kung saan ang mga partisipante ay kumokonekta sa bawat isa sa paraan ng tao-sa-tao na network. Lahat ng mensahe na nakaloob dito ay ipapadala sa pamamagitan ng pagpapahayag. May dalawang uri ng mga tungkulin ang pumiiral: Ordinaryong noda at noda ng bookkeeping. ang mga ordinaryong noda ay ginagamit ang sistema upang makalipat at makipag-palitan, pagtanggap ng mga datos sa talaan; habang ang noda ng bookkeeping ay nagbibigay ng serbisyong akawnting sa pangkalahatang netwok at pinapanatili ang talaan. 
+Ang blockchain ay isang sistema ng ipinamahaging pagtatala na kung saan ang mga partisipante ay kumokonekta sa bawat isa sa paraan ng tao-sa-tao na network. Lahat ng mensahe na nakaloob dito ay ipapadala sa pamamagitan ng pagpapahayag. May dalawang uri ng mga tungkulin ang pumiiral: Ordinaryong node at node ng bookkeeping. ang mga ordinaryong node ay ginagamit ang sistema upang makalipat at makipag-palitan, pagtanggap ng mga datos sa talaan; habang ang node ng bookkeeping ay nagbibigay ng serbisyong akawnting sa pangkalahatang netwok at pinapanatili ang talaan.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: netwok

File: tl-tl/node/whitepaper.md [hunk 4]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
 ###  Pangkalahatang Pamamaraan 
 
 Itakda ang mga agwat ng oras ng paggawa ng bloke bilang t, sa ilalim ng mga normal na pangyayari, ang algoritmo ay magpapatupad ng sumusunod na pamamaraan：
 
-1) Ang pagpapahayag ng transaktsyon na datos ng noda sa panghalahatang network , na nakalakip ang lagda ng nagpadala;
+1) Ang pagpapahayag ng transaktsyon na datos ng node sa panghalahatang network , na nakalakip ang lagda ng nagpadala;
 
-2) Lahat ng mga noda ng bookkeeping ay nagmomonitor ng mga datos ng transaksyon na namamahayag ng malaya at itinatago ang mga datos sa kanyang sariling memorya ayon sa pagkakabanggit; 
+2) Lahat ng mga node ng bookkeeping ay nagmomonitor ng mga datos ng transaksyon na namamahayag ng malaya at itinatago ang mga datos sa kanyang sariling memorya ayon sa pagkakabanggit;
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >nagmomonitor to >nagsusubaybay sa mga datos

File: tl-tl/node/whitepaper.md [hunk 5]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
-Ang mga noda, matapos mamonitor ang pagpapahayag at pagtatanggap ng mga mungkahi, ang magpapatunay ng mga transaksyon. Hindi sila pwedeng sumulat ng bawal na transaksyon sa memorya nang sa huli ay mailantad. Kung ang bawal na transaksyon ay nakapaloob sa iminumungkahi, ang ikot ng konsensus na iyon ay iaabandona at ang pagbabago sa View ay magsisimula agad. Ang pamamaraan ng pagpapatunay ay ang mga sumusunod:
+Ang mga nodw, matapos mamonitor ang pagpapahayag at pagtatanggap ng mga mungkahi, ang magpapatunay ng mga transaksyon. Hindi sila pwedeng sumulat ng bawal na transaksyon sa memorya kung sa huli ay mailantad na. Kung ang bawal na transaksyon ay nakapaloob sa iminumungkahi, ang ikot ng konsensus na iyon ay iaabandona at ang pagbabago sa View ay magsisimula agad. Ang pamamaraan ng pagpapatunay ay ang mga sumusunod:
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >nodw

File: tl-tl/node/whitepaper.md [hunk 6]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
+2) Ang transakyon ba ay nasa blockchain na? Kung oo, ang transaksyon ay pinasiyahang bawal;
 
-3) Are all the contract scripts of the transaction correctly executed? If no, the transaction is ruled illegal;
+3) Ang lahat ba ng iskrip ng kontrata ng transaksyon ay maayos na naipatupad? Kung hindi, ang transaksyon ay pinasiyahang bawal;
 
-4) Is there multiple-spend in the transaction? If yes, the transaction is ruled illegal;
+4) May maramihang paggasta sa transakyon? in the transaction? Kung oo, ang transaksyon ay pinasiyahang bawal;
 
-5) If the transaction had not been ruled illegal in the above procedures, it will be ruled legal;
+5) Kung ang transaksyon ay hindi pinasiyahang bawal sa mga nabanggit na pamamaraan sa itaas, Iyong ay pinasiyahang bawal;
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: ito ay ipinasiyang legal

File: tl-tl/node/whitepaper.md [hunk 7]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
 ### View Change 
 
-If, after 2<sup>𝑣+1</sup> ⋅ 𝑡 time interval, the nodes i cannot reach a consensus or should they receive proposals that contain illegal transactions, the View Change will take place: 
+kung, matapos ng 2<sup>𝑣+1</sup> ⋅ 𝑡 na pagitan ng oras, ang mga node i ay hindi makakaabot ng konsensus o kaya naman ay mabigyan ng mga panukala ng naglalaman ng bawal na mga transaksyon, ang View Change ang kukuha sa pwesto: 
 
-1) Given k=1, 𝑣<sub>𝑘</sub> = 𝑣 + 𝑘 ;
+1) Bigay ang k=1, 𝑣<sub>𝑘</sub> = 𝑣 + 𝑘 ;
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >Given
to 
>Kung may

File: tl-tl/node/whitepaper.md [hunk 8]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
-4) Is there multiple-spend in the transaction? If yes, the transaction is ruled illegal;
+4) May maramihang paggasta sa transakyon? in the transaction? Kung oo, ang transaksyon ay pinasiyahang bawal;
 
-5) If the transaction had not been ruled illegal in the above procedures, it will be ruled legal;
+5) Kung ang transaksyon ay hindi pinasiyahang bawal sa mga nabanggit na pamamaraan sa itaas, Iyong ay pinasiyahang bawal;
 
 ### View Change 
 
-If, after 2<sup>𝑣+1</sup> ⋅ 𝑡 time interval, the nodes i cannot reach a consensus or should they receive proposals that contain illegal transactions, the View Change will take place: 
+kung, matapos ng 2<sup>𝑣+1</sup> ⋅ 𝑡 na pagitan ng oras, ang mga node i ay hindi makakaabot ng konsensus o kaya naman ay mabigyan ng mga panukala ng naglalaman ng bawal na mga transaksyon, ang View Change ang kukuha sa pwesto:
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >Kung

File: tl-tl/node/whitepaper.md [hunk 9]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
-3) Once any node receives at least 𝑛 − 𝑓  same v<sub>k</sub> from different i, the View Change is
-completed. Set 𝑣 = 𝑣<sub>𝑘</sub> and the consensus making begins;
+3) Sa puntong ang node ay nakatanggap ng hindi bababa sa 𝑛 − 𝑓  kagaya sa v<sub>k</sub> sa ibang i, ang View Change ay tapos na. I-set ang 𝑣 = 𝑣<sub>𝑘</sub> at ang paggawa ng konsensus ay magsisimula na;
 
-4) If, after 2<sup>𝑣+1</sup> ⋅ 𝑡 time interval, the View Change is not completed, the k will increase and back to step 2);
+4) Kung, matapos ang 2<sup>𝑣+1</sup> ⋅ 𝑡 na pagitan ng oras, ang View Change ay hindi natapos, ang k ay magdaragdag at babalik sa ikalawang hakbang;
 
-With the k increasing, the overtime waiting time will increase exponentially, so frequent View Change will be avoided and nodes are urged to reach consistency over v.
+Sa patuloy na pagtaas ng k , ang overtaym na oras ng paghihintay ay tataas sa paraang exponensyal, kaya malagiang View Change ay maiiwasang at ang mga node ay hinihimok na maabot ang walang pagbago sa ilalim ng v.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >maiiwasan at

File: tl-tl/node/whitepaper.md [hunk 10]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
-Before the completion of View Change, the original View v is still valid, so unnecessary View Change caused by occasional network latency can be avoided. 
+Bago matapos ang View Change, ang orihinal na View v ay mabisa pa rin, para ang di kailangan na View Change na sanhi ng okasyon na pagbagal ng network ay maiiwasan. 
 
-### Flow Chart
+### tsart ng daloy
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >Tsart ng Daloy

File: tl-tl/node/whitepaper.md [hunk 11]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 
 ![](~/assets/consensus_flowchart.jpg)
 
-## Fault Tolerance Capacity 
+## Kapasidad ng Toleransiya ng kamalian
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >Kamalian

File: tl-tl/node/whitepaper.md [hunk 12]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
 ![](~/assets/consensus_flowchart.jpg)
 
-## Fault Tolerance Capacity 
+## Kapasidad ng Toleransiya ng kamalian 
 
-​Our algorithm provides 𝑓 = ⌊ (𝑛−1) / 3 ⌋ fault tolerance to a consensus system that comprises n nodes. This tolerance capacity includes security and usability and is suite for any network environment.
+​Ang ating algoritmo ay nagbibigay ng 𝑓 = ⌊ (𝑛−1) / 3 ⌋  na toleransiya ng kamalian sa isang sistema ng konsensus na binubuo ng mga n na node. Ang kapasidad ng toleransiya ay naglalaman ng seguridad at abilidad na ito ay pwede gamitin na nababagay sa kahit anong mang network na kaapaligiran.
 
-Request data from nodes contain sender signatures, so malicious bookkeeping nodes cannot falsify requests. Instead, they will try to reverse the system status back to the past, forcing the system to fork.
+Ang mga kahilingan ng mga node sa datos ay naglalaman ng lagda ng nagpadala,nang sa ganun ay ang mga malisyosong node ng bookkeeping ay hindi makakapalsipika ng kahilingan. Sa halip, sila ay magtatangka na baliktarin ang estado ng sistema pabalik sa nakaraan, na pwersahang i-form ang sistema.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >nagpadala, nang sa ganun

File: tl-tl/node/whitepaper.md [hunk 13]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
-then reach a second consensus without informing the R2, revoking the consensus with R1.
-​To reach this, it is necessary that |𝑅1| + |𝐹| ≥ 𝑛 − 𝑓 and |𝑅2| + |𝐹| ≥ 𝑛 − 𝑓 .
-In the worst case scenario, |𝐹| = 𝑓​ , i.e. the number of malicious nodes is at the maximum that the system could tolerate the aforementioned relation becomes |𝑅1| ≥ 𝑛 − 2𝑓​ and ​|𝑅2| ≥ 𝑛 − 2𝑓. Added together, |𝑅1| + |𝑅2| ≥ 2𝑛 − 4𝑓​, which could be simplified as ​𝑛 ≤ 3𝑓. Given that 𝑓 = ⌊ (𝑛−1) / 3 ⌋, which contradicts with the former, it can be proven that the system cannot be forked within the tolerance range. 
+Gayunpaman, sa kapaligiran ng network ng sistema, ang mga node ng konsensus aay nakahati sa tatlong parte: 𝑅 = 𝑅1 ∪ 𝑅2 ∪ 𝐹 , at 𝑅1 ∩ 𝑅2 = ∅ , 𝑅1 ∩ 𝐹 = ∅ ，𝑅2 ∩ 𝐹 = ∅ . Gayunpaman din,
+kapwa R1 at R2 ay mapagkakatiwalaan na mga node ng bookkeeping sa imbakan ng impormasyon na sila lang ang pwedeng makipag-usap sa mga node ng kanilang pangkat;Ang F ay mga malisyosong node sa koordinasyon; bukod dito, ang kondisyong ng network ng F ay nagpapahintulot sa kanila na makipag-usap sa kahit na anong node, kasaali ang R1 at R2.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >pangkat; Ang F

File: tl-tl/node/whitepaper.md [hunk 14]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
+Kung si F ay nagnanais na i-fork ang sistema, sila ay kailangan na umabot sa konsensus kasama ang R1 at maglathala ng mga bloke, at pagkatapos ay maabot ang pangalawaang konsensus ng hindi ipinapaalam kay R2, na pinapawalang bisa ang konsensus kasama ang R1.
+​Para maabot ito, kailangan ay |𝑅1| + |𝐹| ≥ 𝑛 − 𝑓 at |𝑅2| + |𝐹| ≥ 𝑛 − 𝑓 .
+Sa di inaasahang pagkakataon, |𝐹| = 𝑓​ , e.g. ang bilang ng mga malisyosong node ay nasa pinakasukdulan sa ang sistema ay di na kayang tiisin ang nasabing relasyon ay magiging |𝑅1| ≥ 𝑛 − 2𝑓​ at ​|𝑅2| ≥ 𝑛 − 2𝑓. Pagsamahin ang dalawa, |𝑅1| + |𝑅2| ≥ 2𝑛 − 4𝑓​, na pwedeng mapadali kagaya ng ​𝑛 ≤ 3𝑓. Kung ang binigay ay 𝑓 = ⌊ (𝑛−1) / 3 ⌋, na sumasalungat sa dati, ito ay mapapatunayan na ang sistema ay hindi pwedeng i-form forked sa loob ng saklaw ng toleransiya. 
 
-## Reference 
+## Reperensiya 
 
-[1] Nakamoto S. Bitcoin: A peer-to-peer electronic cash system[J]. 2008. 
+[1] Nakamoto S. Bitcoin: tao-sa-tao na na de koryenteng sistema ng peram[J]. 2008.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: peram?

File: tl-tl/node/whitepaper.md [hunk 15]
```diff
@@ -6,113 +6,111 @@ erik@vcage.com
...
+Sa di inaasahang pagkakataon, |𝐹| = 𝑓​ , e.g. ang bilang ng mga malisyosong node ay nasa pinakasukdulan sa ang sistema ay di na kayang tiisin ang nasabing relasyon ay magiging |𝑅1| ≥ 𝑛 − 2𝑓​ at ​|𝑅2| ≥ 𝑛 − 2𝑓. Pagsamahin ang dalawa, |𝑅1| + |𝑅2| ≥ 2𝑛 − 4𝑓​, na pwedeng mapadali kagaya ng ​𝑛 ≤ 3𝑓. Kung ang binigay ay 𝑓 = ⌊ (𝑛−1) / 3 ⌋, na sumasalungat sa dati, ito ay mapapatunayan na ang sistema ay hindi pwedeng i-form forked sa loob ng saklaw ng toleransiya. 
 
-## Reference 
+## Reperensiya 
 
-[1] Nakamoto S. Bitcoin: A peer-to-peer electronic cash system[J]. 2008. 
+[1] Nakamoto S. Bitcoin: tao-sa-tao na na de koryenteng sistema ng peram[J]. 2008. 
 
-[2] Lamport L, Shostak R, Pease M. The Byzantine generals problem[J]. ACM Transactions on Programming Languages and Systems (TOPLAS), 1982, 4(3): 382-401. 
+[2] Lamport L, Shostak R, Pease M. Ang Byzantine na problema ng heneral[J]. ACM na mga transaksyon sa mga lenggwaheng para sa pagprogram at mga sistema (TOPLAS), 1982, 4(3): 382-401.
```
Comment (severinolorillajr) [inline tl-tl/node/whitepaper.md]: >Ang problema sa mga heneral ng Byzantine
>Transkasyon sa mga Lenggwaheng para sa Pag-program at mga Sistema

**Comment by severinolorillajr** [comment]
@agentzero1 , please find a way how to make the comparison to do the review. The history is there, you just have to look for it.

@cordeta , reviewed your PR, make sure to update all of them, then we can merge this.
Thanks.

**Comment by agentzero1** [comment]
@severinolorillajr I'm so sorry

**Comment by cordeta** [comment]
updated all the necessary changes .
Thank You

**Pull request merged**